operate/client/src/modules/request/index.ts

```diff
@@ -19,7 +19,7 @@ type RequestParams = {
 };
 
 async function request({url, method, body, headers, signal}: RequestParams) {
-  const csrfToken = sessionStorage.getItem('OPERATE-X-CSRF-TOKEN');
+  const csrfToken = sessionStorage.getItem('X-CSRF-TOKEN');
```

Comment:
Hi @pedesen ,

In the C8 single app, both Operate and Tasklist will share the same session. 
On the backend, the Operate security package will be used (in the future, it will be replaced by an Identity-provided security module). 
As a result, the backend will consistently return a single CSRF token computed by the Operate security module. 
This CSRF token will be used by both the Tasklist and Operate frontends. 
Therefore, I am renaming the token to a standard name that covers both cases where Operate runs either as a standalone application or within a single app with Tasklist.

Reply:
Okay, makes sense. Is it also planned to add some kind of middleware so that not every app needs to handle CSRF individually?

Reply:
Yes, I think it makes sense to have the CSRF prevention in the security `middleware` that will be provided by Identity (in the coming releases 🤞 )
can you confirm this @romansmirnov and @Ben-Sheppard ?

Reply:
That's how Identity works, so we have one "security" layer and not multiple per app.